### PR TITLE
hs7/chaikin-oscillator

### PR DESCRIPTION
**Pull request by wchmiel**

### CODE REVIEW

File: js/indicators/chaikin.src.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            SPE = EMA.prototype.getValues.call(this, ADL, {
+                period: periods[0]
+            });
+
+            LPE = EMA.prototype.getValues.call(this, ADL, {
+                period: periods[1]
+            });
+
+            // Check if ema is calculated properly, if not skip
+            if (!SPE || !LPE) {
```
Comment (sebastianbochan) [inline js/indicators/chaikin.src.js]: How about verifing ADL values?

File: js/indicators/chaikin.src.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                return false;
+            }
+
+            periodsOffset = periods[1] - periods[0];
```
Comment (sebastianbochan) [inline js/indicators/chaikin.src.js]: How about situation when user set periods: [10,3] ?

File: samples/unit-tests/indicator-chaikin/recalculations/demo.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
+
```
Comment (sebastianbochan) [inline samples/unit-tests/indicator-chaikin/recalculations/demo.js]: In a few places, you use chart.series[0], chart.series[1], chart.series[2]. Consider to use variables.

File: js/indicators/chaikin.src.js (hunk 2 above)
Comment (wchmiel) [reply js/indicators/chaikin.src.js]: There is an if statement above that check it: _line 70_ `if (periods.length !== 2 || periods[1] <= periods[0])`.

**Comment by sebastianbochan** [comment]
Now is fine, so we can merge.

**Pull request merged**